proxy/src/main/java/net/md_5/bungee/UserConnection.java

```diff
@@ -468,22 +491,7 @@ public void sendMessage(ChatMessageType position, BaseComponent... message)
             }
         } else
         {
-            sendMessage( position, ComponentSerializer.toString( message ) );
-        }
-    }
-
-    @Override
-    public void sendMessage(ChatMessageType position, BaseComponent message)
-    {
-        message = ChatComponentTransformer.getInstance().transform( this, true, message )[0];
-
-        // Action bar doesn't display the new JSON formattings, legacy works - send it using this for now
```

Comment:
?? Where did all this code go

Reply:
This method was basically the same as the one with the BaseComponent var args parameter. But it was missing the code to send the action bar with a title packet in 1.11+. All sendMessage methods now call the same final method.

Reply:
For reference the both methods:
https://github.com/SpigotMC/BungeeCord/blob/master/proxy/src/main/java/net/md_5/bungee/UserConnection.java#L450-L488

Reply:
I just looked at the original Mojang ticket for why the Bungee code had been circumvented and apparently the root is fixed in 1.16.2. It might be worth changing that whole block to look something like this:
```
// ref https://bugs.mojang.com/browse/MC-119145
// Versions pre 1.11 cannot send the Action bar with the new JSON formattings
// Work around this by using a Chat packet converting the colors to legacy
if ( getPendingConnection().getVersion() < MINECRAFT_1_11 )
{
  sendMessage( position, ComponentSerializer.toString( new TextComponent( BaseComponent.toLegacyText( message ) ) ) );
// Versions between 1.11 & 1.16.2 still do not support JSON in Chat packet but during this period Mojang introduced
// Title packet and this works with JSON formattings, use this as a workaround!
} else if ( getPendingConnection().getVersion() < ProtocolConstants.MINECRAFT_1_16_2 )
{
  net.md_5.bungee.protocol.packet.Title title = new net.md_5.bungee.protocol.packet.Title();
  title.setAction( net.md_5.bungee.protocol.packet.Title.Action.ACTIONBAR );
  title.setText( ComponentSerializer.toString( message ) );
  unsafe.sendPacket( title );
+++
// Original Mojang client bug is fixed in 1.16.2, not a problem to send normally!
} else 
{
  sendMessage( position, ComponentSerializer.toString( message ) );
}
+++
```

I haven't tested that and I haven't confirmed the ticket comments but in theory it would work. Probably suitable for another PR too but just thought I'd note here as a reminder.